docs/pages/access-controls/guides/moderated-sessions.mdx

```diff
@@ -233,6 +233,21 @@ comma-separated list of usernames.
 This information can be picked up by a third party integration and may for
 example be used to enable notifications over some external communication system.
 
+## File Transfers
+
+File transfers within moderated sessions are available via the Web UI only. When initiating a file transfer, if the
+current active session requires moderation, a file transfer request will automatically be sent to all current party members.
+
+Both the session originator and the moderator(s) must be present in the Web UI during the file transfer initiation to receive
+the file transfer request notification. Once the file transfer has been requested, all members of the party will be notified
+and prompted with an Approve/Deny dialog.
+
+If a Deny is sent, the file transfer request is immediately removed and all party members are notified.
+
+After enough approvals have been given to satisfy the policy (the same policy to start the session), the file transfer
+will automatically begin.
+
+
```

Comment:
It might also make sense to have an FAQ or troubleshooting page with what happens when you try to `tsh scp` when moderation is required.

Reply:
Good idea, will add!

Reply:
having some screen shots would be an added bonus.

Reply:
Are file approve/denies logged as audit log entries?  good to note if they are.

Reply:
Not yet, have an upcoming PR that will add the audit logs and will update the docs then. The file transfer itself is still audited as usual